mern-starter-writeup-v2.md

```diff
@@ -0,0 +1,366 @@
+<br/>We're using the Creator Relationship Management (CRM) App, a full-stack web application designed for influencers, podcasters, and creators. It efficiently manages and showcases client relationships with features like client data management, responsive design, search capabilities, interaction tracking, and project management.
+
+App utilizes MongoDB for data management, Node.js and Express.js for the server-side framework, and React.js with Tailwind CSS for a dynamic user interface.
+
+#### **Setting up MongoDB Cluster**
+
+To set up MongoDB, refer to the [MongoDB Atlas Getting Started guide](https://www.mongodb.com/docs/atlas/getting-started/). This guide will walk you through creating a MongoDB cluster, getting the Mongo URI, and adding your IP address to the network access settings. Once you have your Mongo URI, you can integrate it into your project configuration.
+
+#### **Setting up Daytona for Development Workspace**
+
+Setting up a local development environment can be painful due to version mismatches and configuration issues. [Daytona](https://www.daytona.io/) automatically sets up a Node.js environment with necessary tools, manages port forwarding seamlessly, and runs essential post-create commands, ensuring a consistent, ready-to-use setup that minimizes manual configuration.
+
+To set up Daytona, refer to the [Daytona installation documentation](https://www.daytona.io/docs/installation/installation/). This guide provides detailed steps to get Daytona up and running efficiently, ensuring your development environment is well-configured.
+
+#### **Cloning and Setting Up Your MERN Stack Environment:**
+
+Step 1: Initiate Daytona Server:
+
+- Begin by initiating the Daytona server with the following command:
+
+```
+daytona serve
+```
+Step 2: Create a Workspace
+
+- Initiate the process by executing the following command using this Git URL for the repository:
+```
+daytona create https://github.com/daytonaio-experiments/starter-mern-saas.git
+```
+Step 3: Set a Preferred IDE
+
+- To set up your preferred IDE, list all supported IDEs using the following command and select from them.
+```
+daytona ide
+```
+
+Step 4: Open the Workspace
+
+- Once you've selected your preferred IDE, open the Workspace by executing the following command. Select the workspace created above for this project, and it will open in your chosen IDE.
+```
+daytona code
+```
+
+After following these commands, the repository will open in your preferred IDE.
+
+![workspace open ss](https://github.com/user-attachments/assets/b28d8782-b820-40a4-b892-49dc393d1058)
+
+Step 5: Starting the Backend Server
+
+- Create a .env file in the backend directory and add the following line in it. Change "your_database_connection_string" with your MongoDB connection URI.
+```
+MONGO_URI="your_database_connection_string"
+```
+
+- Start the backend server by running:
+
+```
+npm run dev
+```
+
```

Comment:
- I have changed this server message and made a new pr for the changes: https://github.com/daytonaio-experiments/starter-mern-saas/pull/9#issue-2415534891
- The mflix data you are referring to is sample data created by MongoDB on new collection creation and this will not impact our application in any way. We can remove it manually in our MongoDB dashboard.